src/transport.ts

```diff
@@ -45,16 +46,31 @@ export const CERTHASH_CODE: number = 466
 // @TODO(ddimaria): seems like an unnessary abstraction, consider removing
 export interface WebRTCTransportComponents {
   peerId: PeerId
+  metrics?: Metrics
```

Comment:
@lightsofapollo now that there is more than one attribute to this interface, can you remove my `todo`?

Reply:
resolved